eng/targets/CSharp.Common.targets

```diff
@@ -6,6 +6,30 @@
     <TargetFrameworkIdentifier>.NETFramework</TargetFrameworkIdentifier>
   </PropertyGroup>
 
+  <!-- Ensure API changes show up clearly in PRs. -->
+  <PropertyGroup>
+    <_TFMDirectory>$(TargetFramework)</_TFMDirectory>
+    <_TFMDirectory Condition=" '$(TargetFramework)' == '$(DefaultNetCoreTargetFramework)' ">netcoreapp</_TFMDirectory>
+
+    <!-- Public members should not use oblivious types. Not done with all nullable annotations. -->
+    <NoWarn>$(NoWarn);RS0041</NoWarn>
```

Comment:
@pranavkm where are we on our nullable annotations journey❔

Reply:
Still some ways to go. But you could help: https://github.com/dotnet/aspnetcore/pull/23475

https://giphy.com/gifs/2wG65JG3UAKpW/html5